mne/report.py

```diff
@@ -1130,6 +1156,22 @@ def _render_toc(self, verbose=None):
                             global_id += 1
                         html_toc += u'</ul></li>'
 
+                    if fname.endswith('(whitened)'):
+                        # XXX: remove redundant read_evokeds
```

Comment:
This shouldn't really matter too much, reading evokeds is fast. If it makes the code uglier to re-use, I wouldn't bother.